datasets/mediumComplex.csv

```diff
@@ -5,19 +5,23 @@ Pickup Community Area,Dropoff Community Area,Fare,Trip Seconds,Trip Miles,
 8,24,9,1256,2.23,
 8,56,5,1800,4,
 56,3,11,829,3.79,
-56,32,30,357,16.75,
+56,32,3,357,16.75,

```

Comment:
why are these added?  did i forget to add them is that why or did you add these values?

Reply:
yes i added them because i noticed they were missing for some reason